app/assets/javascripts/TortoiseJS/agent/drawshape.coffee

```diff
@@ -8,16 +8,15 @@ class window.ShapeDrawer
   setTransparency: (ctx, turtleColor) ->
     ctx.globalAlpha = if turtleColor.length > 3 then turtleColor[3] / 255 else 1
 
-
-  drawShape: (ctx, turtleColor, shapeName) ->
+  drawShape: (ctx, turtleColor, shapeName, thickness = 1) ->
```

Comment:
Should `turtleColor` still be called "turtleColor" even though it's no longer just the turtle's color.